docs/next.config.js

```diff
@@ -31,6 +31,9 @@ module.exports = {
   eslint: {
     ignoreDuringBuilds: true,
   },
+  experimental: {
+    runtime: 'nodejs',
```

Comment:
Agree, we don't really need it, I've added it just to be on the safe side in case if it changes, following https://github.com/mui/material-ui/pull/33196#issuecomment-1159762620

Reply:
Done with https://github.com/mui/material-ui/pull/33196/commits/33114b31f31d9cab3d2fdb0dc4558bca37158bf4 It doesn't exists anymore.